anaconda.spec.in

```diff
@@ -187,6 +187,8 @@ Requires: tmux
 Requires: gdb
 # support for installation from image and live & live image installations
 Requires: rsync
+# Submitting bugs to bugzilla
+Requires: libreport-rhel-anaconda-bugzilla
```

Comment:
Shouldn't it be a dependency of `install-img-deps`?

Reply:
I am not really sure, actually. Do we have the Workstation variant in RHEL 9? If not, then this should indeed go to the img package.

Reply:
RHEL have a Server with GUI which is basically workstation. However, it's still not Live media if you asking on this.